database/seed.js

```diff
@@ -0,0 +1,100 @@
+const faker = require('faker');
+const {connection} = require('./db.js');
+
+// Table types
+function generateTypes() {
+
+  function getProducts() {
+    // var products = [];
+    // for (var i = 0; i < 30; i++) {
+    //   products.push([faker.commerce.product()]);
+    // }
+    // return products;
+
+    var set = new Set();
+    while (set.size < 5) {
+      set.add(faker.commerce.product());
+    }
+
+    var arr = Array.from(set);
+    return arr.map(type => [type]);
+
+  }
+
+  var values = getProducts();
+  // console.log(values);
+
```

Comment:
Perhaps use some more descriptive variable names for what the set/arr, etc. represent?